src/testRunner/parallel/worker.ts

```diff
@@ -338,4 +340,5 @@ export function start() {
     }
 
     process.on("message", processHostMessage);
+    const importTestsPromise = importTests();

```

Comment:
This looks weird because it appears to be an unused `const`. Can we move these two lines above the function declarations?

Reply:
Yeah, I'll move this and then force an early return for clarity.

Reply:
Oh, it's not so simple; there's code right above here...

Reply:
The best I think I can do is to make a `let` var at the top and then silence the lint rule that says that there's only one assignment.

Reply:
Came up with something better, PTAL